strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/repository/RepositoryEvent.java

```diff
@@ -0,0 +1,8 @@
+package org.carlspring.strongbox.repository;
+
+public enum RepositoryEvent
```

Comment:
may be misleading with org.carlspring.strongbox.event.repository.RepositoryEvent

Reply:
+1: There's already a class in `strongbox-event-api`. Do we need a separate event listener for searches, or maybe you could just add it to the existing class?

Reply:
right
removed
sorry for this garbage
thanks